docs/config.md

```diff
@@ -286,3 +288,62 @@ During some heavy workload cloud, the time to create and delete openstack instan
 you can set:
 `CLUSTER_API_OPENSTACK_INSTANCE_DELETE_TIMEOUT` for instance delete timeout value.
 `CLUSTER_API_OPENSTACK_INSTANCE_CREATE_TIMEOUT` for instance create timeout value.
+
+## Use machineset as additional worker nodes
```

Comment:
I think we should use `machinedeployment` instead of `machineset` everywhere to be more consistent.